crates/shared/src/http_solver.rs

```diff
@@ -158,8 +161,6 @@ impl HttpSolverApi for DefaultHttpSolverApi {
                 .append_pair("auction_id", auction_id.to_string().as_str());
         }
         let query = url.query().map(ToString::to_string).unwrap_or_default();
-        let body = serde_json::to_string(&model).context("failed to encode body")?;
-        tracing::trace!(%url, %body, "request");
```

Comment:
Removed because with the new code we don't always have a string to log. We could serialize the json twice so that we can log it but I don't think it's useful enough to keep.